src/otx/core/config/hpo.py

```diff
@@ -38,3 +38,4 @@ class HpoConfig:
     asynchronous_bracket: bool = True
     asynchronous_sha: bool = num_workers > 1
     metric_name: str | None = None
+    adapt_bs_search_space_max_val: Literal["None", "Safe", "Full"] = "None"
```

Comment:
Isn't mixing uppercase and lowercase letters unnatural from a user usability perspective?

Reply:
This feature uses 'auto batch size' feature, so I think it's better to align its naming to avoid confusion.